gradle/libs.versions.toml

```
@@ -1,22 +1,22 @@
 [versions]
 animalSniffer = "1.5.4"
-arrowGradleConfig = "0.10.1"
-assertj = "3.22.0"
+arrowGradleConfig = "0.10.2"
```

```suggestion
arrowGradleConfig = "0.10.1"
```
this one needs to be `0.10.1`, as of now,

ref: https://github.com/arrow-kt/arrow/pull/2738